src/main/config/config.c

```diff
@@ -418,6 +420,16 @@ static void resetConf(void)
     featureSet(DEFAULT_FEATURES);
 #endif
 
+#ifdef OSD
+    resetOsdConfig();
+#endif
+
+#ifdef USE_RTC6705
+    featureSet(FEATURE_VTX);
+    masterConfig.vtx_channel = 19; // default to Boscam E channel 4
+    masterConfig.vtx_power   = 1;
+#endif
```

Comment:
You don't need `featureSet(FEATURE_VTX)` instead `FEATURE_VTX` should be added to `DEFAULT_FEATURES` in the `target.h` file. 

Can you also move the default settings elsewhere, since we are trying to avoid additions to `config.c`